src/main.cc

```diff
@@ -663,47 +663,55 @@ static void parse_command_line_for_debug_option(gint argc, gchar *argv[])
 #endif
 }
 
-#if HAVE_CLUTTER
-static gboolean parse_command_line_for_clutter_option(gint argc, gchar *argv[])
+static gboolean search_command_line_for_option(const gint argc, const gchar* const argv[], const gchar* option_name)
 {
-	const gchar *clutter_option = "--disable-clutter";
-	gint len = strlen(clutter_option);
-	gboolean ret = FALSE;
+	const gint name_len = strlen(option_name);
 
-	if (argc > 1)
+	for (gint i = 1; i < argc; i++)
 		{
-		gint i;
-
-		for (i = 1; i < argc; i++)
+		const gchar *current_arg = argv[i];
+		// TODO(xsdg): This actually only checks prefixes.  We should
+		// probably replace this with strcmp, since strlen already has
+		// the shortcomings of strcmp (as compared to strncmp).
+		//
+		// That said, people may be unknowingly relying on the lenience
+		// of this parsing strategy, so that's also something to consider.
+		if (strncmp(current_arg, option_name, name_len) == 0)
 			{
-			const gchar *cmd_line = argv[i];
-			if (strncmp(cmd_line, clutter_option, len) == 0)
-				{
-				ret = TRUE;
-				}
+			return TRUE;
 			}
 		}
 
-	return ret;
+	return FALSE;
+}
+
+static gboolean search_command_line_for_unit_test_option(gint argc, gchar *argv[])
+{
+	return search_command_line_for_option(argc, argv, "--run_unit_tests");
```

Comment:
I guess convention for option format is kebab-case. Would you mind to rename it to `--run-unit-tests`?

Reply:
https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html
"Option names are typically one to three words long, with hyphens to separate words"

Reply:
Fixed; thanks